Cross Region VM Restore Points.md

```diff
@@ -0,0 +1,396 @@
+# Cross-Region-VM-Restore-Points
```

Comment:
All the comments added in this PR are applicable to the Api Documentation file too. Pls. make updates to that file as well.